git_machete/client.py

```diff
@@ -219,7 +219,8 @@ def add(self,
             opt_onto: Optional[LocalBranchShortName],
             opt_as_root: bool,
             opt_yes: bool,
-            verbose: bool = True
+            verbose: bool = True,
+            switch_branch: bool = True
```

Comment:
The current name will be misleading, as the switch only happens when the branch is **newly created**, not when it's already existed before (lines 264 onwards)

Reply:
Okay!

Reply:
Or even: `switch_head_if_new_branch`, to make it clear that it's not always that a new branch will be created